status/emoji.go

```diff
@@ -0,0 +1,33 @@
+package status
+
+func Emoji(input PRInfo) []string {
+	if input.Mergeable == MergeableState_Mergeable && (input.Review == ReviewState_Approved || input.Review == ReviewState_NoReviewRequired) {
+		return []string{"🟢"}
+	}
+
+	emoji := []string{"🔴"}
+
+	switch input.Mergeable {
+	case MergeableState_Mergeable:
+		break
+	case MergeableState_Conflict:
+		emoji = append(emoji, "🚧")
+	case MergeableState_Unknown:
+		emoji = append(emoji, "❓")
```

Comment:
I so want this out of code and into configuration xD

Reply:
We can change the emoji into something else if you prefer. But I can't see them really changing their meaning in the near future. Do you think it's important for the user to be able to remap these emoji?